mongodb_store/scripts/mongodb_play.py

```diff
@@ -171,6 +179,12 @@ def run(self, running):
                 else:
                     delay = publish_time - now
                     rospy.sleep(delay)
+                if action_goal_match:
+                    float_secs = time.time()
+                    secs = int(float_secs)
+                    nsecs = int((float_secs - secs) * 1000000000)
```

Comment:
explain why these values are set, the `1000000000`

Reply:
did 0 work?

Reply:
This was copied from rospy on how they get the wall clock time.

Reply:
I added a comment saying where I got these lines of code frm